static/app/utils/discover/fieldRenderers.tsx

```diff
@@ -337,6 +338,17 @@ const SPECIAL_FIELDS: SpecialFields = {
       );
     },
   },
+  replayId: {
+    sortField: 'replayId',
+    renderFunc: data => {
+      const replayId: string | unknown = data?.replayId;
```

Comment:
```suggestion
      const replayId = data?.replayId;
```

Nit: We shouldn't need to specify types on this variable because the next line does the `typeof` refinement and we learn that the value is either a string or not.